src/basic/parse-util.c

```diff
@@ -732,3 +732,40 @@ int parse_loadavg_fixed_point(const char *s, loadavg_t *ret) {
 
         return store_loadavg_fixed_point(i, f, ret);
 }
+
+int named_pcr(char *pcr, unsigned *n){
+        if (streq(pcr, "corefirmware"))
```

Comment:
rather than a list of if/else, create a table and use `DEFINE_STRING_TABLE_LOOKUP` to get accessors

Reply:
On it sir.

Reply:
> rather than a list of if/else, create a table and use `DEFINE_STRING_TABLE_LOOKUP` to get accessors

I created a table and used a for loop sir. 
Is it okay to not exactly use the DEFINE_STRING_TABLE_LOOKUP macro?


Also, I've been going through the log messages, I'm not able to trace the cause of the 4 test failures.
(CentOS CI)

Reply:
> use DEFINE_STRING_TABLE_LOOKUP, there's no point in doing that by hand

Alright sir. Let me add that knowledge to myself.